.buildkite/pipelines/pull_request/security_solution.yml

```diff
@@ -5,7 +5,7 @@ steps:
       queue: n2-4-spot
     depends_on: build
     timeout_in_minutes: 120
-    parallelism: 15
+    parallelism: 10
```

Comment:
@patrykkopycinski FYI

Reply:
How's this number being chosen?

Reply:
it's just a temporary step, we're working on a solution that will automatically pick the number based on the previous runs